index.html

```diff
@@ -3388,21 +3388,21 @@ <h3>Media Types</h3>
 (or a shortened version) in the media type may or may not imply a proof. The
 Working Group is seeking implementation experience related to the use of media
 types and the expectation related to whether or not a particular media type
-is expected to be secure with a particular proof.
-        </p>
-        <p class="issue" data-number="1065">
-The VCWG is currently debating whether all media types that start with
-`application/vc+` indicate an expectation around the securing mechanism or not.
-Working Group is seeking implementation experience related to the use of media
-types and the expectation related to whether or not a particular media type
 is expected to be secure with a particular proof.
         </p>
         <p>
 There is one media type associated with the core data model:
 `application/vc+ld+json`. Other specifications, such as [[?VC-JWT]],
 define other media types that contain transformation rules that allow for
 those other syntaxes to be transformed to the `application/vc+ld+json`
-media type.
+media type. Media types that allow for transformation into
+`application/vc+ld+json` SHOULD begin with `application/vc+`.
+        </p>
+        <p>
+The media type `application/vc+ld+json` does not imply any particular securing
+mechanism. It describes an object that is securable. Other specifications
+defining media types that begin with `application/vc+` SHOULD NOT define those
+media types to imply any particular securing mechanism.
```

Comment:
I had to read this last sentence a number of times and I'm still not sure I understand what it's saying.

I think it's saying: "This core spec doesn't imply a securing mechanism, and other media types shouldn't imply a securing mechanism either"... 

IOW, "that other thing might have `proof` and `experimentalSecuringMechanism` and be wrapped in a CWT (or any combination thereof), so... you can't presume any of those things... you're just going to have to verify it using your own checks to see if it truly is verifiable, and then validate it to make sure it meets your business requirements."

I don't know how to say that more succinctly... once we have confirmation on the intent, we can probably go from there.

As an aside, it sounds like we're starting to collect: "Normative statements for specification authors that are authoring other specifications that contain transformation rules to `application/vc+ld+json`."

Reply:
I agree, I would remove this:

```
Other specifications
defining media types that begin with `application/vc+` SHOULD NOT define those
media types to imply any particular securing mechanism.Other specifications
defining media types that begin with `application/vc+` SHOULD NOT define those
media types to imply any particular securing mechanism.
```

A general statement about media types here might be:

Implementers are advised not assume veracity or security of content regardless of the media type used to communicate it. A validation process needs to be applied to confirm integrity, media types should not be trusted, or assumed to be sure without confirming by some validation process.

Reply:
This has been refactored in [0d5dbd9](https://github.com/w3c/vc-data-model/pull/1107/commits/0d5dbd985a84c27311cc86225a898904f115c1af)